recipes/graphene/all/conandata.yml

```diff
@@ -1,7 +1,4 @@
 sources:
-  "1.10.2":
```

Comment:
Any reason to drop this?

Reply:
Not specifically. Build was taking a while and there were already 4 minor versions. I've restored it.